src/graphql/schemas/resolvers/userResolver.ts

```diff
@@ -14,6 +25,29 @@ class UserResolver extends UserBaseResolver {
   async users(@Args() args: GetUserArgs) {
     return this.getUsers(args);
   }
+
+  @FieldResolver(() => [SignatureRequest])
+  async signature_requests(@Root() user: User) {
+    if (!user.address) {
+      return [];
+    }
+    const args: GetSignatureRequestArgs = {
+      where: {
+        safe_address: {
+          eq: user.address,
+        },
+      },
+    };
+
+    const response = await this.supabaseDataService.getSignatureRequests(args);
```

Comment:
As this inherits the BaseResolver, I think the pattern should match the fetching pattern as you see in the `hypercertResolver`. In other works, the command would be `return await this.getSignatureRequests(args)`